neo4j/observation.go

```diff
@@ -95,7 +95,7 @@ func (n *Neo4j) InsertObservationBatch(ctx context.Context, attempt int, instanc
 			return errors.Wrap(err, "observation batch save failed")
 		}
 
-		log.Event(ctx, "got an error when saving observations, attempting to retry", log.WARN, log.Data{
+		log.Warn(ctx, "got an error when saving observations, attempting to retry", log.Data{
```

Comment:
I would include error in warning log: 
```suggestion
		log.Warn(ctx, "got an error when saving observations, attempting to retry", log.FormatErrors([]error{err}), log.Data{
```

Reply:
good shout - added here, and will keep an eye out for this in the future